{
    "type": "bundle",
    "id": "bundle--c017a025-2ab2-40c2-ada2-a72451ddd731",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--582f899b-d8d9-4f4c-9349-fb0d57b4436d",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-06-06T00:29:46.988805Z",
            "modified": "2024-06-06T00:29:46.988805Z",
            "name": "CVE-2024-5629",
            "description": "An out-of-bounds read in the 'bson' module of PyMongo 4.6.2 or earlier allows deserialization of malformed BSON provided by a Server to raise an exception which may contain arbitrary application memory.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-5629"
                }
            ]
        }
    ]
}